### DISCUSSION TITLE
Local Network Service Access

### DISCUSSION THREAD

**Discussion by ralphshep** [ConSol/docker-headless-vnc-container#183] [Q&A]
Hello,
I'm using this app to remotely access and manage my TrueNAS Scale server, its services, and its apps. So far it has worked great for accessing the local web UIs for NextCloud, TrueNAS UI, and PhotoPrism using their local IP address. However, I also want to mount the SMB and WebDAV network shares to the Thunar File Manager (or alternative) but it appears this feature has been disabled. For reference I'm using the consol/debian-xfce-vnc

Thanks!

**Comment by sni** [comment]
i moved this into discussions, i don't think this is an issue and more like a question.
